src/components/DebugPanel.tsx

```
@@ -63,50 +63,43 @@ export const DebugPanel: Component<DebugPanelProps> = (props) => {
   return (
     <div
-      class="bg-white border-t border-slate-200 text-[10px] font-mono text-slate-600 flex overflow-hidden shrink-0 transition-colors duration-300 selection:bg-primary/10 selection:text-primary z-20 relative"
+      class="bg-[var(--color-earthy-bg)] border-t border-[var(--color-earthy-sage)] text-[10px] font-mono text-[var(--color-earthy-dark-brown)] flex overflow-hidden shrink-0 transition-colors duration-300 selection:bg-[var(--color-earthy-coral)]/20 selection:text-[var(--color-earthy-coral)] z-20 relative"
       style={{ height: `${height()}px` }}
     >
       {/* Resize Handle */}
       <div
-        class="absolute top-0 left-0 w-full h-1 cursor-ns-resize z-50 hover:bg-primary/50 transition-colors bg-transparent"
+        class="absolute top-0 left-0 w-full h-1 cursor-ns-resize z-50 hover:bg-[var(--color-earthy-muted-green)]/50 transition-colors bg-transparent"
         onMouseDown={handleMouseDown}
       />
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)